index.js

```diff
@@ -76,6 +76,13 @@ module.exports = {
     this.jshintrc = app.options.jshintrc;
   },
 
+  postBuild: function () {
```

Comment:
I think that having both ember-cli-qunit AND ember-cli-mocha is an error condition and continuing is unlikely to go well...

We can also remove `ember-cli-qunit` in [the blueprint](https://github.com/switchfly/ember-cli-mocha/blob/master/blueprints/ember-cli-mocha/index.js) to make this even more unusual of a case...